docs/source/enable_cc_lifecycle.md

```diff
@@ -12,7 +12,7 @@ Updating a channel configuration is, at a high level, a three step process (for
 
 We will be performing these channel configuration updates by leveraging a file called `enable_lifecycle.json`, which contains all of the updates we will be making in the channel configurations. Note that in a production setting it is likely that multiple users would be making these channel update requests. However, for the sake of simplicity, we are presenting all of the updates as how they would appear in a single file.
 
-Note: this topic describes a network that does not use a "system channel", a channel that the ordering service is bootstrapped with and the ordering service exclusively controls. Since the release of v2.3, using a system channel is now considered the legacy process as compared to the new process to [Create a channel](./create_channel/create_channel_participation.html) without a system channel. For a version of this topic that includes information about the system channel, check out [Enabling the new chaincode lifecycle](https://hyperledger-fabric.readthedocs.io/en/release-2.2/enable_cc_lifecycle.html) from the v2.2 documentation.
+Note: this topic describes a network that does not use a "system channel" since it is no longer supported from release v3.0 onwards. Please refer to [Create a channel](./create_channel/create_channel_participation.html) for more details.
```

Comment:
Again, I don't think a Note is needed here at all, since there is no other option now.